source/mainnet/smart-contracts/tutorials/nft-minting/setup-dev-env.rst

```diff
@@ -0,0 +1,108 @@
+.. include:: ../../../variables.rst
+.. _setup-dev-env:
+
+=================================
+Setup the development environment
+=================================
+
+First, you need to install “rustup” which installs Rust and Cargo to your computer. To do that, you can use this command.
+
+.. code-block:: console
+
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+Select **1** to continue the installation.
+
+.. image:: images/mint-install-rustup.png
+    :width: 100%
+
+Finally, when Rust and Cargo are successfully installed in your system, you should see something similar to below.
+
+.. image:: images/mint-rust-install-done.png
+    :width: 100%
+
+Copy and paste the commands in a terminal to install Wasm which will be used for building contracts.
+
+.. code-block:: console
+
+    source “$HOME/.cargo/env”
+    rustup target add wasm32-unknown-unknown
```

Comment:
I am not sure why the `source` is needed. Rustup installation should take care of that.